Library/Formula/miruo.rb

```diff
@@ -0,0 +1,15 @@
+class Miruo < Formula
+  homepage "https://github.com/KLab/miruo/"
+  url "https://github.com/KLab/miruo/archive/0.9.6a.tar.gz"
+  sha256 "346e8bd892fe15e6859486d50231238879d3b620b93df444b13a910e84864e7f"
+  version "0.9.6a"
+
+  def install
+    system "./configure", "--prefix=#{prefix}", "--disable-dependency-tracking", "--with-libpcap=#{MacOS.sdk_path}/usr"
+    system "make", "install"
+  end
+
+  test do
+    system "#{sbin}/miruo", "--version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!